api/src/main/resources/messages.properties

```diff
@@ -494,10 +494,10 @@ coreapps.units.meters=m
 coreapps.units.centimeters=cm
 coreapps.units.millimeters=mm
 coreapps.units.kilograms=kg
-coreapps.units.degreesCelsius=°C
```

Comment:
What change is this?

Reply:
i think that came when i ran the code-formatter as per opemrs standards

Reply:
i seem not able to see the actaul change , its neither a space nor a new line , cant see the actual change